src/mwindow.c

```diff
@@ -226,7 +222,7 @@ static bool git_mwindow_scan_recently_used(
 		}
 	}
 
-	if (!lru_window && !lru_last)
+	if (!lru_window)
```

Comment:
maybe instead here check whether `lru_last` was modified within the loop.

Reply:
This uses a flag now